### ISSUE TITLE
Please support blob 

### ISSUE THREAD

**Issue by imobicloud** [AndreaVitale/av.imageview#2]
```
<ImageView id="photo" class="av-imageview" module="av.imageview"/>
$.photo.image = blob;
```

Pass blob to image property cause error

```
message = "Invalid type passed to function";
nativeLocation = "-[AvImageviewImageView setImage_:] (AvImageviewImageView.m:105)";
nativeReason = "expected: String, was: TiBlob";
```

thank you :)

**Comment by imobicloud** [comment]
Android does not load Local images, show the brokenLinkImage image

".photo-source": { width: 200, height: 200, borderRadius: 100, touchEnabled: false }
".av-imageview": { defaultImage: '/images/photo-default.jpg', brokenLinkImage: '/images/photo-broken.jpg' }

<ImageView id="photo" class="photo-source av-imageview" module="av.imageview"/>

$.photo.image = '/images/onboard/upload-client.png';

It works on iOS

**Comment by AndreaVitale** [comment]
About blob support it was solved with [819b8c8](https://github.com/AndreaVitale/imageview/commit/819b8c8adffc3ea394250b9d8aef13997f350be0).

About local images for me are working also on Android. Try with latest module version and if is still not working please open another issue with a  reproducible test case.
